src/commands/tasks/util.ts

```diff
@@ -25,3 +30,154 @@ export async function zipFileFromDirectory(path: string): Promise<string> {
     archive.finalize();
   });
 }
+
+export async function createFunctionUser(sdk: OAuth1Client, data: { taskName: string; targetEmail?: string; targetPermissions: string[]; }) {
```

Comment:
(silly) Might also be nice to call this something like "ensure"  rather then "create"